Tinylog/ViewModels/TasksViewModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TasksViewModel.swift
+//  Tinylog
+//
+//  Created by Spiros Gerokostas on 19/7/20.
+//  Copyright © 2020 Spiros Gerokostas. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+final class TasksViewModel {
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/c61f36452fa17d076822a3b73830cc49c60091be/Tinylog/ViewModels/TasksViewModel.swift#L13"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TasksViewModel.swift:13</code></td>
    </tr>
  </tbody>
</table>